docs/test/checkbox/ds4/index.html

```diff
@@ -101,10 +109,10 @@ <h2>Custom Icon</h2>
     </span>
 </div>
 
-<h2>Custom Icon Disabled</h2>
+<h2>Custom Icon, Disabled</h2>
 <div class="test">
     <span class="checkbox" style="color: #5ba71b">
-        <input aria-label="Default checkbox example" class="checkbox__custom-control" disabled type="checkbox" value="1" name="test3b" />
+        <input aria-label="Default checkbox example" class="checkbox__control" disabled checked type="checkbox" value="1" name="test3b" />
```

Comment:
Is there a reason it became checked?

Reply:
Good question. I just think the disabled example looks better when checked. That's all really. Let me know if I've overlooked something.